watershed_workflow/river_mesh.py

```diff
@@ -760,6 +769,45 @@ def convexity_enforcement(river, corr, widths, dilation_width, gid_shift):
     return shapely.geometry.Polygon(corr_coords_new)
 
 
+from scipy.spatial import ConvexHull
```

Comment:
please put imports at the top, or within the function if you don't want to require the package.  Is scipy in our current environment?

Reply:
yes, scipy  is in our WW environment